$MirOS: ports/multimedia/mplayer/patches/patch-libmpdemux_open_c,v 1.1.7.1 2005/03/18 15:50:49 tg Exp $
--- libmpdemux/open.c.orig	Sun Feb 27 04:24:50 2005
+++ libmpdemux/open.c	Thu May 19 20:32:06 2005
@@ -562,7 +562,7 @@ if(strncmp("dvd://",filename,6) == 0){
 }
 
 int dvd_parse_chapter_range(m_option_t *conf, const char *range){
-  const char *s;
+  char *s;
   char *t;
   if (!range)
     return M_OPT_MISSING_PARAM;
